recipe/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "wasmtime" %}
+{% set version = "11.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
So these are just the required components that you put into specific folders?

Reply:
they are git submodules which are part of the project but not included in tarball